packages/ember-metal/lib/properties.js

```diff
@@ -148,6 +149,7 @@ if (EMBER_METAL_ES5_GETTERS) {
       });
 
       trap.toString = trap.valueOf = () => '[COMPUTED PROPERTY]';
+      trap.toJSON = trap.valueOf = () => '[COMPUTED PROPERTY]';
 
```

Comment:
This should be folded into the previous line

Reply:
Sorry. I did't understand the above statement.
Do you mean something like below?
```
trap.toString = trap.toSJON = trap.valueOf = () => '[COMPUTED PROPERTY]';
```
I can raise a PR, if needed.

Reply:
Yes, that’s what Godfrey was suggesting

Reply:
Raised a PR for this #16365